packages/devextreme/testing/tests/DevExpress.ui.widgets/listParts/commonTests.js

```diff
@@ -234,10 +239,11 @@ QUnit.module('collapsible groups', moduleSetup, () => {
         }).dxList('instance');
 
         const $items = this.element.find(toSelector(LIST_ITEM_CLASS));
+        const firstItemIsFocused = $items.eq(0).hasClass(`${LIST_FOCUS_STATE_CLASS}`);
```

Comment:
Is it okay that we store it before we expand the group? Seems strange.
Same in other tests